src/main/java/com/github/mauricioaniche/ck/Runner.java

```diff
@@ -7,7 +7,9 @@
 import java.util.Map;
 
 public class Runner {
-
+	public static Map<String,Integer> methodOccurrences = new HashMap<>();
```

Comment:
The `Runner` should ideally have no business logic at all. You should move this somewhere else.